source/features/confirm-too-short-comments.tsx

```diff
@@ -0,0 +1,37 @@
+// import select from 'select-dom';
+import delegate from 'delegate-it';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+
+
+function handleIssueComment(event: delegate.Event<MouseEvent, KeyboardEvent, HTMLAnchorElement>): void {
+    let body = document.querySelector('textarea[name="comment[body]"], textarea#issue_body')?.value.length;
+	let title = document.querySelector('input#issue_title')?.value.length;
+	if ( ((body && body < 3) || (title && title < 3)) && !confirm('Less than 3 chrs, are you sure?') ) {
+			event.preventDefault();
+			event.stopImmediatePropagation(); // I don't know if it's necessary to stop comments, they're sent via ajax
```

Comment:
This comment is for you to find out, not to be part of the code

Reply:
Ok, removed the comment.

Reply:
Is the code necessary? Did you try without?

Reply:
Yes, it is: I see it's necessary for blocking comments submission.